src/docs/components/docs-main/docs-main.css

```diff
@@ -213,8 +215,12 @@
       border: 0;
       height: var(--ld-sp-1);
 
-      &:last-of-type + p {
+      &:last-of-type {
         display: none;
+
+        & + p {
+          display: none;
+        }
```

Comment:
That's not necessary, is it?
```suggestion
```

Reply:
It is, in the docs of the tabs component.

Reply:
How can something inside a `display: none;`-container ever be visible?

Reply:
Ok, read it wrong ^^"